metals/src/main/scala/scala/meta/internal/metals/newScalaFile/NewFileTypes.scala

```diff
@@ -9,6 +9,11 @@ object NewFileTypes {
     def toQuickPickItem: MetalsQuickPickItem = MetalsQuickPickItem(id, label)
   }
 
+  case object JustPackage extends NewFileType {
+    override val id: String = "just-package"
+    override val label: String = "Empty file with package"
+  }
```

Comment:
Maybe it would be ok to name it just as `ScalaFile` / `id = scala-file / `label = Scala file`?
It should work similarly no new file creation as it's currently works (package is added automatically) with only difference that you don't need to type `.scala`